spec/models/user_spec.rb

```diff
@@ -265,4 +314,28 @@
       expect(user).to_not have_admin_access
     end
   end
+
+  describe '#is_admin?' do
+    let(:user) { create(:user, roles: [role_name]) }
+
+    subject { user.is_admin? }
+    context 'with role hr' do
+      let(:role_name) { :hr }
+      it 'is considered an admin' do
+        is_expected.to eq(true)
+      end
+    end
+    context 'with role admin' do
+      let(:role_name) { :admin }
+      it 'is considered an admin' do
+        is_expected.to eq(true)
+      end
+    end
+    context 'with role super admin' do
+      let(:role_name) { :super_admin }
+      it 'is considered an admin' do
+        is_expected.to eq(true)
```

Comment:
`expect(subject.is_admin?).to be_truthy` or `expect(user.is_admin?).to be_truthy`